conference.js

```diff
@@ -1283,6 +1283,27 @@ export default {
             // FIXME close
         });
 
+        room.on(ConferenceEvents.SUSPEND_DETECTED, () => {
+            // after wake up, we will be in a state where conference is left
+            // there will be dialog shown to user, and new page need to be
+            // navigated, so we do not care about device change, which happens
+            // on resume after suspending PC
+            if (this.deviceChangeListener)
+                JitsiMeetJS.mediaDevices.removeEventListener(
+                    JitsiMeetJS.events.mediaDevices.DEVICE_LIST_CHANGED,
+                    this.deviceChangeListener);
+
+            // stop local video
+            if (localVideo) {
+                // this calls room.removeTrack internally
+                // so we don't need to remove it manually
+                localVideo.dispose().then(() => {
```

Comment:
Why showing the overlay depends on disposing the local video? Also if we are going to dispose the video track, lets dispose the audio as well.